aiohttp_swagger/__init__.py

```diff
@@ -46,7 +46,8 @@ def setup_swagger(app: web.Application,
                   contact: str = "",
                   swagger_home_decor: FunctionType = None,
                   swagger_def_decor: FunctionType = None,
-                  swagger_info: dict = None):
+                  swagger_info: dict = None,
```

Comment:
Add a few lines of documentation and example how it could be done. Probably add tests too.